packages/datagateway-search/src/searchPageTable.component.tsx

```diff
@@ -117,9 +131,32 @@ function a11yProps(index: string): React.ReactFragment {
 
 const StyledBadge = withStyles(badgeStyles)(Badge);
 const StyledTabs = withStyles(tabStyles)(Tabs);
+const StyledBox = withStyles(boxStyles)(Box);
+
+const ViewCart = (props: SearchCartProps): React.ReactElement => {
+  const [t] = useTranslation();
+  return (
+    <div>
+      <IconButton
+        className="tour-dataview-cart-icon"
+        onClick={props.navigateToDownload}
+        aria-label={t('searchPageTable.cart_arialabel')}
+      >
+        <Badge
+          badgeContent={
+            props.cartItems.length > 0 ? props.cartItems.length : null
+          }
+          color="primary"
+        >
+          <ShoppingCartIcon />
+        </Badge>
+      </IconButton>
+    </div>
+  );
+};
```

Comment:
This code is repeated - can we extract it into it's own tiny component file to reduce code repetition?

Reply:
ok, I'll moving this into datagateway-common